extensions/email/src/main/java/org/mule/extension/email/api/retriever/RetrieverConnection.java

```diff
@@ -68,7 +68,15 @@ public RetrieverConnection(EmailProtocol protocol,
         try
         {
             this.store = session.getStore(protocol.getName());
-            this.store.connect(username, password);
+
+            if (username != null && password != null)
+            {
+                this.store.connect(username, password);
```

Comment:
the config allows for specifying a username but not a password. AFAIK, that's also possible on terribly badly configure servers.